components/service/pocket/src/main/java/mozilla/components/service/pocket/stories/db/PocketLocalStory.kt

```diff
@@ -0,0 +1,31 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.service.pocket.stories.db
+
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+/**
+ * A Pocket recommended story that is to be mapped to SQLite table.
+ */
+@Entity(tableName = PocketRecommendationsDatabase.TABLE_NAME_STORIES)
+internal data class PocketLocalStory(
```

Comment:
"Local" meaning here that this is the database object, right? I think in other places we named them `*Entity` (`DownloadEntity`, `CrashEntity`, `TabCollectionEntity`, ..). But I am also okay if we want to deviate here?

Reply:
Tried to find a system to tie `PocketLocalStory` closely to `PocketApiStory`.
Entity seem like the already recognized pattern. Will go with that, thank you!